PeerioTechnologies/peerio-desktop
# WIP: UserSearchError with separate whitelabel component

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/6891/medcryptor-desktop-doctor-patient-invite-screen-in-the-contacts-tab

#### Testing instructions  
This PR adds a UserSearchError component which is used in the UserPicker component (in NewChat and NewChannel) and NewContact views. (There are entirely separate files for Peerio and MedCryptor.) Activate it by searching for a username or email that does not correspond to an existing user.

* Peerio:
  * in NewChat or NewChannel, both username and email search should not be any different from current behaviour (on `dev` or `prod`)
  * in NewContact, both username and email search should use the new UserSearchError component, so it should be consistent with NewChat/NewChannel appearance and behaviour
* MedCryptor:
  * username search is unchanged from Peerio
  * in NewChannel, email search does not prompt user to send an invite "Send" button
  * in NewChat or NewContact, email search prompts user to send an invite with selector for doctor/admin or patient invite

In NewContact view the mockups show a different styling for "Add or invite a contact". I don't know if that is within the scope of this task; it seems to reflect a broader update to header typography that hasn't yet been passed to development.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
